src/python/WMCore/Services/WMStatsServer/WMStatsServer.py

```diff
@@ -65,6 +65,15 @@ def _createQuery(self, queryDict):
 
         return args.rstrip('&')
 
+    def getGlobalLocks(self):
+        """
```

Comment:
How about elaborating on this docstring? Specifying the return format is also useful IMO.

Reply:
Done